tests/plugins/dataRefs.test.ts

```diff
@@ -186,4 +186,55 @@ describe('DataRefs plugin', () => {
             });
         });
     });
+
+    describe('data with strings', () => {
+        const y = new Yagr(window.document.createElement('div'), {
+            timeline: [1, 2, 3, 4],
+            scales: {
+                y: {stacking: true},
+            },
+            series: [
+                {data: [1, 'x', 3, 4], scale: 'y', id: 'one'},
+                {data: [2, 'x', null, 3], scale: 'y', id: 'two'},
```

Comment:
Could you explain whats the difference between 'x' and null values?

Reply:
https://yagr.tech/en/api/data-processing#problem